src/components/Link.react.js

```diff
@@ -55,6 +55,16 @@ export default class Link extends Component {
 
     render() {
         const {className, style, id, href, loading_state} = this.props;
+
+        const setChildren = () => {
+            if (
+                this.props.children === null ||
+                typeof this.props.children === 'undefined'
+            ) {
```

Comment:
There's a Ramda helper [`isNil`](https://ramdajs.com/docs/#isNil) that does exactly that - you can probably inline this whole thing below using it:
`{isNil(children) ? href : children}`

Reply:
Perfect, exactly what I was looking for, I think I'm going to look at using some more Ramda functions for more efficient code. Added in 263984a.